functests/4_latency/latency.go

```diff
@@ -255,6 +240,71 @@ var _ = Describe("[performance] Latency Test", func() {
 	})
 })
 
+//read LATENCY_TEST_RUN environment var and parse it if set
```

Comment:
minor: this is not a good godoc comment - but it's not relevant because the function is not exported. A good godoc comment should be in the form
```
// Foobar does...
func Foobar() {}
```
note the comment starts with the same name as the function

Reply:
thanks for the info will consider it next time. I will remove it.

Reply:
no need, let's consider next time.

Reply:
got that !